### DISCUSSION TITLE
CMS - Dienste - Themen(Eigenschaften) - v.5.22.4103

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#11] [Q&A] [answered]
Aus einem Dienst mit 20+ Layern soll nur 1 Layer in die Layer-Schaltungen übernommen werden. Die restlichen werden über Themen(Eigenschaften) deaktiviert (unsichtbar, gesperrt). 
Wie kann man das möglichst schnell (ohne viel mühsame Klick-Arbeit) erreichen?
JO

**Comment by gis-eni** [comment, marked as answer]
Das ist. so.
Der empfohlene Weg ist, dann man die Dienste schon so aufbereitet, dass man möglichst wenig im CMS noch nachbessern muss. Baut man beispielsweise den TOC im Dienst schon richtig zusammen muss man sich im CMS nicht mehr um die Darstellungsvarianten kümmern.
Je weiter weg die eigentliche Dienst von der Präsentation im WebGIS ist, je mehr Klickarbeit hat man.
